### PR TITLE
Building arm64 image

### LINKED ISSUE #264: Build for linux/arm64 (multi arch)
We're using the ARM based instances in AWS (graviton), and we can't run the aiven operator on those machines as you're only building for `linux/amd64`. 

It would be great if you could start building multi arch images. It won't require a huge effort on you end, just a few changes to the Dockerfile and the GH actions workflow.

https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

### PR DESCRIPTION
**Pull request by christoffer-eide**
Fixes #264

Note that I have not tested the GH actions workflow.

### CODE REVIEW

**Comment by Serpentiel** [comment]
hey 👋 

thanks for your contribution! I will check the PR now

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -43,6 +43,9 @@ jobs:
 
     - id: action_vars
       run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+    - uses: docker/setup-qemu-action@v1
+      with:
+        platforms: arm64
```
Comment (Serpentiel) [inline .github/workflows/release.yaml:48]: doesn't this breaks `amd64` build then?

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (christoffer-eide) [reply .github/workflows/release.yaml:48]: Don't think so as the GH actions runners run on `amd64`. I could add `amd64` just to be safe? 
The whole GH actions build would need to be tested by you guys

Comment (christoffer-eide) [reply .github/workflows/release.yaml:48]: I'm not 100% sure if quemu is needed in the first place

Comment (Serpentiel) [reply .github/workflows/release.yaml:48]: ahh, looks like this whole `with` block [is not needed](https://www.winters.nz/docker/cross-platform-github-actions/), could you please remove it?

Comment (Serpentiel) [reply .github/workflows/release.yaml]: thanks, we'll see if it works or not :)

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**